modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -130,9 +130,21 @@ export async function deleteRelationship(id: number) {
 		});
 	}
 	catch (error) {
-		Liferay.Util.openToast({
-			message: (error as Error).message,
-			type: 'danger',
+		const errorMessage = (error as Error).message;
+		Liferay.Util.openModal({
+			bodyHTML: `<p>${errorMessage}</p>`,
+			buttons: [
+				{
+					displayType: 'warning',
+					label: Liferay.Language.get('done'),
```

Comment:
@larissacribeiro I'm not sure the label "done" is the best one to be used here. It should be used in a context where the user is performing an action. Please check this with a designer.

Reply:
@naraandrade I understand, however I've spoken to the designer and concluded that since this is the current pattern in the portal, we should maintain it in order to avoid different approaches to the same matter, and after we can open an issue to correct this message in all warning modals throughout the portal.

Reply:
@larissacribeiro Ok, thanks for checking that!